### PR TITLE
:art: rename intent

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: src/components/footer/footer.container.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const mapStateToProps = (state) => {
 export const mapDispatchToProps = (dispatch) => {
   return {
     goTo: href => () => dispatch(push(href)),
-    onClearCompleted: () => dispatch(action('ON_CLEAR_COMPLETED')),
+    onClearCompleted: () => dispatch(action('CLICK_CLEAR_COMPLETED')),
```
Comment (fabienjuif) [inline src/components/footer/footer.container.js]: We should stick with `on`
ON_CLICK_CLEAR_COMPLETED

**Review by fabienjuif — CHANGES_REQUESTED** [review]
Stick with `ON`

File: src/components/todos/todos.container.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const mapStateToProps = (state) => {
 
 export const mapDispatchToProps = (dispatch) => {
   return {
-    onCompleteAll: e => dispatch(action('ON_COMPLETE_ALL', e.target.checked)),
+    onCompleteAll: e => dispatch(action('COMPLETE_ALL_CHANGED', e.target.checked)),
```
Comment (fabienjuif) [inline src/components/todos/todos.container.js:16]: clicked ? swaped ?

File: src/sagas/intents.js [hunk 3]
```diff
@@ -9,19 +9,19 @@ const ESCAPE_KEY = 27
...
+  yield takeLatest('TODO_DOUBLECLICKED', action => ui.edit(action.payload))
+  yield takeLatest('TODO_BLURED', action => todos.update(action.payload))
+  yield takeLatest('TODO_CHANGED', action => ui.setEditTodo(action.payload))
+  yield takeLatest('CLEAR_COMPLETED_CLICKED', todos.clearCompleted)
+  yield takeLatest('NEW_TODO_KEYDOWNED', (action) => {
     if (checkEnter(action)) return todos.add(action.payload.todo)
     return null
   })
-  yield takeLatest('ON_KEYDOWN_TODO', (action) => {
+  yield takeLatest('TODO_KEYDOWNED', (action) => {
```
Comment (fabienjuif) [inline src/sagas/intents.js]: keydowned ? is it English ?

Comment (guillaumecrespel) [reply src/sagas/intents.js]: yes yes !

**Review by fabienjuif — APPROVED** [review]

**Pull request merged**